src/components/pages/search/search-category-result/SearchCategoryResult.tsx

```
@@ -0,0 +1,156 @@
+                      durationTime={item?.duration_ms}
+                      uri={item.uri}
+                      key={`track-${idx}`}
+                    />
+                  ))}
+              </div>
+            </div>
+          </div>
+          {Object.keys(data)
+            .filter((key) => key !== "tracks")
+            .map((key: string, idx: number) => (
+              <div className={cx("search-category-result-section", key)}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)